src/Scrollbar.react.js

```diff
@@ -465,7 +470,7 @@ var Scrollbar = React.createClass({
   },
 
   _blur() {
-    if (this.isMounted()) {
+    if (this._isMounted {
```

Comment:
```js
if (this.isMissingAClosingParen {
  throw new Error('Ah crap')
}
```

:)

Reply:
Thought to myself, "Could've sworn I caught and fixed that..." and I was right, I just forgot to push. Fixed now!